packages/flutter/lib/src/gestures/tap_and_drag.dart

```diff
@@ -1087,6 +1107,8 @@ sealed class BaseTapAndDragGestureRecognizer extends OneSequenceGestureRecognize
       _pastSlopTolerance = _pastSlopTolerance || _getGlobalDistance(event, _initialPosition) > computedSlop;
 
       if (_dragState == _DragState.accepted) {
+        final OffsetPair delta = _calculateDragUpdateDelta(event);
```

Comment:
Is it necessary to calculate the drag update delta here? Can we use `event.position`, and `event.localPosition` directly instead?

Reply:
Not sure if the positions are valid for the details, but I'll double-check if your suggestion works.

Reply:
This looks like what `LongPressEndDetails` and `DragEndDetails` do for the most part.

Reply:
Further question about the transform: What is the initial idea about the transform calculation to the event?

Reply:
Are you referring to the logic in `_acceptDrag`?

Reply:
Yep, it used to be the only one that corrects the positions.

Reply:
I think that one can remain. It's mainly to handle `DragStartBehavior` iirc.

Reply:
Done update